frontend/components/atoms/Button/index.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+
+const Button = ({ title, type }) => {
+  switch (type) {
+    case "submit":
+      return (
+        <button
+          type={type}
+          className="items-center px-5 py-2.5 text-sm font-medium text-center text-red-700 rounded-lg border-2 border-red-500 focus:ring-1 focus:ring-red-600 hover:bg-rose-200 "
+        >
+          {title}
+        </button>
+      );
+      break;
+    default:
+      return;
+      break;
+  }
+};

```

Comment:
```suggestion
const Button = ({ usage= "primary", type = "submit", additionalClass, isDisabled = false, children, onClick }) => {
  const getClass = () => {
    if(usage === "primary"){
      return `bg-blue-600 py-2 hover:bg-blue-800`
    }
  }
  
  return (
    button 
            onClick={onClick}
            disabled={isDisabled}
            type={type}
            className={`text-white font-bold py-2 rounded-lg ${getClass()} ${additionalClass}`}
    >
            {children}
    </button>
  )
 
}

```
You have to identify the common classes that are used for the buttons and put them in the className. Then for the unique classes for usage such as Primary, Secondary, Success buttons you can put then in the getClass function